plugins/builds/artifacts/get.js

```diff
@@ -7,6 +7,7 @@ const uuid = require('uuid/v4');
 
 const idSchema = joi.reach(schema.models.build.base, 'id');
 const artifactSchema = joi.string().label('Artifact Name');
+const DOWNLOAD_TYPE = joi.string().valid(['', 'true']).label('Flag to trigger download');
```

Comment:
Just wondering if `false` is valid as well?

Reply:
will add